atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/cassandra/util/CassandraCommands.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.ete.cassandra.util;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.common.io.BaseEncoding;
+import com.palantir.atlasdb.ete.EteSetup;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+
+public class CassandraCommands {
+    public static void nodetoolFlush(String containerName) throws IOException, InterruptedException {
+        EteSetup.execCliCommand(containerName, "nodetool flush");
+    }
+
+    public static List<String> nodetoolGetSSTables(String containerName,
+            String keyspace,
+            TableReference tableRef,
+            byte[] rowKey) throws IOException, InterruptedException {
+        String query = String.format("nodetool getsstables %s %s__%s %s",
```

Comment:
is there a utility that converts a tablespace to %s__%s? We must have done this conversion when creating the table.

Reply:
Yep - `CassandraKeyValueServiceImpl.internalTableName(tableRef)` should do it, changed to use that instead.